backend/tests/integration/records/views/test_views_literature.py

```diff
@@ -407,29 +409,16 @@ def test_literature_facets_author_count_returns_non_empty_bucket(
 
 
 def test_literature_facets_arxiv(api_client, db, es_clear, create_record):
-    record = create_record("lit")
+    create_record("lit")
     response = api_client.get("/literature/facets")
     response_data = json.loads(response.data)
     response_status_code = response.status_code
-    response_data_facet_keys = list(response_data["aggregations"].keys())
     response_data_hits = response_data["hits"]["hits"]
 
     expected_status_code = 200
     expected_data_hits_source = {}
-    expected_facet_keys = [
-        "arxiv_categories",
-        "author",
-        "author_count",
-        "doc_type",
-        "earliest_date",
-        "subject",
-        "collaboration",
-    ]
-    expected_facet_keys.sort()
-    response_data_facet_keys.sort()
 
     assert expected_status_code == response_status_code
-    assert expected_facet_keys == response_data_facet_keys
```

Comment:
why?

Reply:
It was a leftover. But now that I looked into the test, it didn't actually make sense to me. It seems to be testing the exact same thing as [test_literature_facets|https://github.com/inspirehep/inspirehep/blob/fe96ade1a8ca881747a2c56ef55c9dfd4fbb5de3/backend/tests/integration/records/views/test_views_literature.py#L328]